src/js/pages/search/getRecipesByIngredient.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Created by Fran on 29/10/2017.
+ */
+
+// Dependencias del proyecto
+const { getState } = require("../../appState");
+/**
+ * Funcio que recibe un ingrediente por parametro y devuelve todas las recetas que lo
+ * contienen.
+ * @param ingredient
+ * @returns {Array}
+ */
+function getRecipesByIngredients(ingredient) {
+	const recetas = getState().recipes;
+
+	const result = [];
+	for (let i = 0; i < recetas.length; i += 1) {
```

Comment:
Para iterar un array te recomiendo un [`for..of`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of), los `for...i` son muy del siglo XX jajaja

Está bien, si quieres no tienes por qué cambiarlo, es una recomendación.

Reply:
![eslint warning](https://user-images.githubusercontent.com/23703546/32291977-ec4475c2-bf3e-11e7-97fd-b1c6c6eae4a1.png) 

Lo intente hacer con un for of pero me daba este warning el eslint

Reply:
Sí... yo también me acabo de encontrar con ese mismo problema en el commit 221a35a. Por lo visto la guía de estilo de AirBNB, que es la que usamos, no lo permite.

Lo he solucionado usando `Array.forEach`, que también es preferible a `for..i`. En cualquier caso, hazlo de la forma que tú quieras, esto tampoco tiene mucha importancia.